packages/web-app-files/src/App.vue

```diff
@@ -61,7 +65,15 @@ export default defineComponent({
       closeSidebar: 'close',
       setActiveSidebarPanel: 'setActivePanel'
     }),
-
+    onDragEnter(event) {
+      event.stopPropagation()
```

Comment:
Can you try `@dragenter.stop.prevent` in the template? That should do the same and is IMO a bit cleaner because you narrow down the event subscription at its root.